packages/base/tests/functional/basics/Environment.lua

```diff
@@ -379,6 +379,32 @@ id     string [env]
 		env:createPlacement{max = 200}
 
 		unitTest:assertEquals(#cs.cells[1]:getAgents(), #predators)
+		predators = Society{
+			instance = predator,
+			quantity = 100
+		}
+
+		cs = CellularSpace{xdim = 10}
+		env = Environment{cs, predators}
+		env:createPlacement{strategy = "spread"}
```

Comment:
Add another test with `max = 3` similar to this one. Verify if each cell contains at most three agents and in the end verify if every agent is within a cell.